MySolution.rb

```diff
@@ -0,0 +1,48 @@
+class MySolution
+
+    #problem1
+    def sumOfEven(array)
+        sum = 0
+        for i in array
+            if i.even?
+                sum += i
+            end
+        end
+        return sum
+    end
+    
+    #problem2
+    def getLargest2Numbers(array)
+        bigMax = -1
```

Comment:
The problem states `an array of integers` does not say `unsigned integers`. So this solution will fail for an array like `[-3, -2]`

Reply:
You are right, I must had in mind the hypothesis from problem 3 after I read them. I'll add this Test Case and correct the solution

Reply:
You're right, I've must had in mind the hypothesis from problem 3 after I read the problems. I'll correct it